### PR TITLE
feat(ingestion): powerbi # support platform instance

### PR DESCRIPTION
**Pull request by mohdsiddique**

### CODE REVIEW

File: metadata-ingestion/src/datahub/ingestion/source/powerbi/powerbi.py [hunk 1]
```diff
@@ -878,8 +890,12 @@ def report_to_datahub_work_units(
 @platform_name("PowerBI")
 @config_class(PowerBiDashboardSourceConfig)
 @support_status(SupportStatus.CERTIFIED)
+@capability(SourceCapability.DESCRIPTIONS, "Enabled by default")
+@capability(SourceCapability.PLATFORM_INSTANCE, "Enabled by default")
+@capability(SourceCapability.PLATFORM_INSTANCE, "Enabled by default")
```
Comment (jjoyce0510) [inline metadata-ingestion/src/datahub/ingestion/source/powerbi/powerbi.py:895]: is this duplicated?

**Review by jjoyce0510 — COMMENTED** [review]
What about platform instances in the urns to upstream datasets parsed from M-Query?

**Review by jjoyce0510 — APPROVED** [review]
This works... but we probably also need a connection_to_platform_instance map similar to what we've done for looker

**Review by jjoyce0510 — COMMENTED** [review]
Why does the docs site already indicate there is a config called "platform_instance"?

**Comment by hsheth2** [comment]
@mohdsiddique looks like tests are failing

**Comment by mohdsiddique** [comment]
> @mohdsiddique looks like tests are failing

It was because of master merge, updated golden file for big-query tables

**Review by hsheth2 — APPROVED** [review]

File: metadata-ingestion/src/datahub/ingestion/source/powerbi/powerbi.py [hunk 2]
```diff
@@ -878,8 +890,11 @@ def report_to_datahub_work_units(
 @platform_name("PowerBI")
 @config_class(PowerBiDashboardSourceConfig)
 @support_status(SupportStatus.CERTIFIED)
+@capability(SourceCapability.DESCRIPTIONS, "Enabled by default")
```
Comment (jjoyce0510) [inline metadata-ingestion/src/datahub/ingestion/source/powerbi/powerbi.py:893]: Thank you!

**Pull request merged**